common/bcp47/transform-destination.xml

```diff
@@ -30,6 +30,7 @@
 			<type name="null" description="make no change in the string" since="29"/>
 			<type name="remove" description="remove every character in the string" since="29"/>
 			<type name="zawgyi" description="map Unicode to Zawgyi Myanmar encoding" since="34"/>
+			<type name="morse" description="map Unicode to Morse Code encoding" since="41"/>
```

Comment:
Is the "Morse" part right for this PR or should it be with the other PR?

Reply:
Adding the "morse" identifier addition to this request was mostly a matter of convenience since it was overlooked in the earlier PR.

On Ethiopic-Gemination, it would follow the **S-T** ID pattern and would be formed from the respective attributes from the Ethiopic-Gemination.xml file

`<transform source="Ethiopic" target="Gemination" direction="forward" visibility="internal">`

My guess is that the "Ethiopic-Gemination" identifier was not registered ahead of its first reference by another transliteration file (like Ethiopic-Latin-Aethiopica_Auto_Geminated.xml) which depends on it.  In this case, why the identifier doesn't get registered in time is a mystery.

The approach with the "Preemptive Gemination" files follows the approach used by the "InterIndic" conversions (and others) where "InterIndic" and "Gemination" are just used as an intermediate conversion step (thus the "internal" visibility flag).

Yes, I am open to splitting the PR into smaller PRs.